test/templates/pagination/component.njk

```diff
@@ -0,0 +1,35 @@
+{#
+    see ./data.json for object structure
+#}
+
+{% macro pagination(properties={}, modifier={}, state={}, aria={}) -%}
```

Comment:
This is fine to rough out the elements, but think about building out the logic to render the component based on the data you supply. If you have 2345 items, with 30 per page and you are on page 28, what would you get. This is the best way to test the component — you can pass a variety of values to make you get the proper output.